pandas/core/window/rolling.py

```diff
@@ -1706,10 +1707,12 @@ def validate(self):
 
     def _validate_monotonic(self):
         """
-        Validate on is_monotonic.
+        Validate monotonic (increasing or decreasing).
         """
-        if not self._on.is_monotonic:
-            formatted = self.on or "index"
+        if not (self._on.is_monotonic_increasing or self._on.is_monotonic_decreasing):
```

Comment:
Surprised this doesn't match the previous definition of just `is_monotonic`. Would welcome that as a follow up to align the logic

Reply:
You mean updating `is_monotonic` to return True also for monotonic decreasing? I tried that, but some of its callers rely on the fact that the sequence is also increasing and many tests failed afterwards. So I would rather not do that, especially not in this PR.